stub/storage.go

```diff
@@ -17,6 +17,8 @@ var mx = sync.Mutex{}
 // below represent map[servicename][methodname][]expectations
 type stubMapping map[string]map[string][]storage
 
+type matchFunc func(interface{}, interface{}) bool
```

Comment:
Created a function that accepts two parameters with any types, and returns bool. With this we can do "equals", "contains" and "matches".